src/tree/private_tree.rs

```diff
@@ -50,7 +50,7 @@ impl PrivateTree {
             hpke_private_key: None,
             path_keys: PathKeys::default(),
             commit_secret: CommitSecret {
-                secret: Secret::new_empty_secret(),
+                secret: Secret::default(),
```

Comment:
If you derive `Debug` for `CommitSecret` as well, it's only `CommitSecret::default()` 🙂

Reply:
I assume you mean `Default`? Done.